Sheikh_Shahzeb/User3_Management_System/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+	def index
+		@users =User.all
+	end
+	def new
+
+	end
+	def create
+		User.create(first_name:params[:first_name],last_name:params[:last_name], email_address:params[:email_address],password:params[:password])
+		redirect_to('/')
+	end
+	def show
+		@user = User.find(params[:id])
+	end
+	def edit
+		@user = User.find(params[:id])
```

Comment:
Since you have to get the user multiple times, this would be a good case to use a before_action like this:
     before_action :set_user, only: [:show, :edit, :update, :destroy]
and then you would just have to define the set_user method